tests/components/automation/test_state.py

```diff
@@ -536,9 +537,10 @@ def setup_comp(hass):
     )
     await hass.async_block_till_done()
 
-    assert hass.states.async_remove("test.entity")
+    assert hass.states.async_remove("test.entity", context=context)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5238698)